tests/unit-tests/test_utils.py

```diff
@@ -463,6 +568,26 @@ def test_gen_mrr_score():
 
     assert th.isclose(metrics['mrr'], metrics_opti['mrr'])  # Default tolerance: 1e-08
 
+def test_stream_dist_tensors_to_hdf5():
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        # get the test dummy distributed graph
+        # medium size has 1,000,000 nodes, which is enough (>chunk_size)
+        g, _ = generate_dummy_dist_graph(tmpdirname, size="medium")
+
+        dummy_dist_embeds = {}
+        for ntype in g.ntypes:
+            dummy_dist_embeds[ntype] = DistTensor((g.number_of_nodes(ntype), 5),
+                      dtype=th.float32, name=f'ntype-{ntype}',
+                      part_policy=g.get_node_partition_policy(ntype))
+
+        stream_dist_tensors_to_hdf5(dummy_dist_embeds, os.path.join(tmpdirname, "embed_dict.hdf5"))
```

Comment:
are you sure you trigger the code that writes data in chunks.

Reply:
i suggest you add `chunk_size` as an argument and set the right chunk size to trigger the code.

Reply:
updated as suggested.